tests/ut_eventmodel/eventmodeltest.cpp

```diff
@@ -716,6 +716,8 @@ void EventModelTest::testDeleteEventGroupUpdated()
 
     QVERIFY(groupModel.databaseIO().getGroup(group.id(), group));
 
+    QTest::qWait(1000); // wait for database to update correctly
```

Comment:
Are you sure there's no signal or you can wait for? Or a condition using QTRY_COMPARE?